currencies/ticker_task_test.go

```diff
@@ -0,0 +1,308 @@
+package currencies_test
```

Comment:
The tests in this file were copied from `rate_converter_test.go` and slightly modified. I'm still using sleep here. I think it'll be a fair amount of extra ,not straightforward code to mock the ticker and I wasn't sure if it would be a good use of my time right now, especially since I think we should still have some integration tests anyway and these tests serve that purpose. Open to feedback on this of course.